doc-content/jbpm-docs/src/main/asciidoc/Integration/integration-osgi-con.adoc

```diff
@@ -0,0 +1,12 @@
+[id='integration-osgi-con_{context}']
+= Integration with OSGi
```

Comment:
Again, can you make this more specific so that it is standalone?

Reply:
I am adding a deprecation note to this one, so it is definitely not a candidate for working out more details.